README.md

```diff
@@ -1 +1,5 @@
-# review1
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
It would be a good idea to add a descriptive README file to our project. There are several templates you can refer to around the web. Remember we use our README file to describe to our web app users what we did in our project and what the requirements are to be able to use our project.

Reply:
- [Optional] Please when given a review on the readme, you can add a link to the readme template.